doc/lua_api.txt

```diff
@@ -2496,15 +2496,25 @@ Elements
           (default: infinite).
 
 ### `style[<name>;<prop1>;<prop2>;...]`
+### `style[<name>:<state>;<prop1>;<prop2>;...]`
 
 * Set the style for the named element `name`.
+* <state> is a comma-separated list of states.
```

Comment:
`<state>` is interpreted as HTML. Please use \`state\` like above.
Also in below in `style_type`